src/lib/viewers/controls/box3d/AnimationClipsControl.tsx

```diff
@@ -6,7 +6,7 @@ import './AnimationClipsControl.scss';
 type AnimationClip = {
     duration: number;
     id: string;
-    name: string;
```

Comment:
Can you give some context on the need for this change? We should try to leave the source as-is if we can.

Reply:
Yes, it seems that `name` is a global variable name so it's clashing - https://eslint.org/docs/latest/rules/no-restricted-globals

Reply:
These aren't global variables, though, they're properties of an object. Is this another case of some kind of parser oddity for TypeScript files?

Reply:
Huh, it looks like if I remove `parser: '@babel/eslint-parser'` from the eslint config, this error goes away